cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -58,82 +61,88 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
+        p("include::show-all-privileges-syntax.asciidoc[]")
+        query("SHOW PRIVILEGES", assertPrivilegeShown(Seq(
+          Map("access" -> "GRANTED", "action" -> "access", "role" -> "regularUsers"),
+          Map("access" -> "DENIED", "action" -> "access", "role" -> "noAccessUsers")
+        ))) {
+          p(
+            """Lists all privileges for all roles.
+              |The table contains columns describing the privilege:
+              |
+              |* access: whether the privilege is granted or denied (whitelist or blacklist)
+              |* action: which type of privilege this is: traverse, read, match, write, a database privilege, a dbms privilege or admin
+              |* resource: what type of scope this privilege applies to: the entire dbms, a database, a graph or sub-graph access
+              |* graph: the specific database or graph this privilege applies to
+              |* segment: for sub-graph access control, this describes the scope in terms of labels or relationship types
+              |* role: the role the privilege is granted to
+              |""".stripMargin)
+          resultTable()
+        }
 
-      p("It is also possible to filter and sort the results by using `YIELD`, `ORDER BY` and `WHERE`.")
-      query("SHOW PRIVILEGES YIELD role, access, action ORDER BY action WHERE role = 'admin' ", assertPrivilegeShown(Seq(
-        Map("access" -> "GRANTED", "action" -> "access", "role" -> "admin"),
-        Map("access" -> "GRANTED", "action" -> "write", "role" -> "admin")
-      ))) {
-        p(
-          """In this example:
-            |
-            |* The number of columns returned has been reduced with the `YIELD` clause.
-            |* The order of the returned columns has been changed.
-            |* The results have been filtered to only return the 'admin' role using a `WHERE` clause.
-            |* The results are ordered by the 'action' column using `ORDER BY`.
-            |
-            |`SKIP` and `LIMIT` can also be used to paginate the results.
-            |""".stripMargin)
-        resultTable()
-      }
+        p("It is also possible to filter and sort the results by using `YIELD`, `ORDER BY` and `WHERE`.")
+        query("SHOW PRIVILEGES YIELD role, access, action ORDER BY action WHERE role = 'admin' ", assertPrivilegeShown(Seq(
+          Map("access" -> "GRANTED", "action" -> "access", "role" -> "admin"),
+          Map("access" -> "GRANTED", "action" -> "write", "role" -> "admin")
+        ))) {
+          p(
+            """In this example:
+              |
+              |* The number of columns returned has been reduced with the `YIELD` clause.
+              |* The order of the returned columns has been changed.
+              |* The results have been filtered to only return the 'admin' role using a `WHERE` clause.
+              |* The results are ordered by the 'action' column using `ORDER BY`.
```

Comment:
```suggestion
              |* The results are ordered by the *action* column using `ORDER BY`.
```

Reply:
Why only update `action` and not `admin` on the row above? Also same comment here about * vs `

Reply:
Updated `admin` in the same way you had updated `regularUsers` and `jake` below

Reply:
*action* (or `action`) is the header of a column while `admin` is not. But yes, admin should also be wrapped in `'s.